application-admintools-ui/src/main/resources/AdminTools/Code/Macros.xml

```diff
@@ -152,5 +152,53 @@
     &lt;/div&gt;
   &lt;/div&gt;
 #end
+#macro (pagesOverNumberOfCommentsModal $maxComments)
+  #set ($discard = $xwiki.jsfx.use('js/xwiki/table/tablefilterNsort.js', true))
+  &lt;div class="modal fade" id="pagesOverNumberOfComments" tabindex="-1" role="dialog"
+    aria-labelledby="pagesOverNumberOfCommentsLabel"&gt;
```

Comment:
I don't see this id. You should use it on the title, so to have that being used as the label of this element https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby
https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminGroupsSheet.xml#L42